cmd/ferretdb/main.go

```diff
@@ -125,7 +125,7 @@ var (
 		slog.LevelError.String(),
 	}
 
-	logFormats = []string{"console", "text", "json"}
+	logFormats = []string{"console", "mongo", "text", "json"}
```

Comment:
Is that the best order?

Reply:
Fixed the ordering, so `mongo` is next to the `json` handler.